.github/workflows/other-lint.yml

```diff
@@ -0,0 +1,36 @@
+name: Lint Code Base
+
+on:
+  pull_request:
+    paths-ignore:
```

Comment:
Why paths-ignore and not just white list the correct path?

Reply:
The `SuperLinter` lints a ton of different languages (including ruby, php, etc) so i just ignored the paths that our other linters already cover